s3_deploy.sh

```diff
@@ -1,39 +1,99 @@
 #!/bin/bash
 
-# this will deploy the current public folder to a subfolder in the s3 bucket
-# the subfolder is the name of the TRAVIS_BRANCH
-if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
-	echo "skiping deploy to S3: this is a pull request"
-	exit 0
+# Typically this is the Project name
+# The portal report has its own bucket and deploys to the root of it
+S3_BUCKET_PREFIX=''
+# AWS CloudFront distribution ID
+DISTRIBUTION_ID='E19KBL7INSABNU'
+# AWS CloudFront distribution domain
+DISTRIBUTION_DOMAIN='portal-report.concord.org'
+# name of branch to deploy to root of site
+ROOT_BRANCH='production'
+# Bucket to deploy to, typically this is 'model-resourcs', but some projects
```

Comment:
```suggestion
# Bucket to deploy to, typically this is 'model-resources', but some projects
```
This typo presumably exists in other instances of this file as well. How did I/we miss it before? 🤷

Reply:
Thanks. I just updated it in starter-projects too.